### PR TITLE
Add jaraco as primary owner of importlib.metadata and importlib.resources.

### PR DESCRIPTION
**Pull request by jaraco**

### CODE REVIEW

**Comment by jaraco** [comment]
This change removes @ericsnowcurrently , @encukou , and @ncoghlan from these sub modules and @brettcannon from `importlib.metadata`. Happy to leave you as shared owners. Let me know if you want that.

**Review by brettcannon — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -60,6 +60,8 @@ Python/pythonrun.c            @iritkatriel
 # bytecode.
 **/*import*.c                 @brettcannon @encukou @ericsnowcurrently @ncoghlan @warsaw
 **/*import*.py                @brettcannon @encukou @ericsnowcurrently @ncoghlan @warsaw
+**/importlib/resources/*    @jaraco @warsaw @brettcannon
+**/importlib/metadata/*    @jaraco @warsaw
```
Comment (ezio-melotti) [inline .github/CODEOWNERS]:
```suggestion
**/importlib/resources/*      @jaraco @warsaw @brettcannon
**/importlib/metadata/*       @jaraco @warsaw
```

**Review by warsaw — APPROVED** [review]
Yes, I'm happy to remain a co-owner, even though @jaraco is doing all the heavy lifting these days.

File: .github/CODEOWNERS (hunk 1 above)
Comment (jaraco) [reply .github/CODEOWNERS]: Thanks. Yeah, I couldn't see where they aligned.

**Review by ezio-melotti — APPROVED** [review]

**Comment by jaraco** [comment]
Proceeding with merge. Happy to accept amendments subsequently.

**Pull request merged**